app/controllers/miq_policy_controller.rb

```diff
@@ -943,7 +943,9 @@ def folder_get_info(folder_node)
     @sb[:folder] = nodeid.nil? ? nodetype.split("-").last : nodeid
     if x_active_tree == :policy_tree
       if nodeid.nil? && ["compliance", "control"].include?(nodetype.split('-').last)
-        @folders = ["Host #{nodetype.split('-').last.titleize}", "Vm #{nodetype.split('-').last.titleize}"]
+        @folders = MiqPolicy::UI_FOLDERS.collect do |model|
```

Comment:
@jrafanie If this is indeed textual only (I'm not sure since I could not find where this is used)
then I can add i18n in a different patch (I don't want to do this bugfix here) to the two occurrences you mentioned and [this one](https://github.com/moolitayer/manageiq/blob/container_openscap_policies/app/controllers/miq_policy_controller/policies.rb#L232) 

@h-kataria can you confirm that @folders should be internationalized?